.github/workflows/openqa.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Run a basic openQA test
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+  # Note how this runs on:pull_request_target and not on:pull_request!
+  # The difference is that this runs always with the context of the master
+  # branch. This is necessary to allow accessing the API credential secrets.
+  workflow_dispatch:
+env:
+  OPENQA_HOST: ${{ secrets.OPENQA_URL }}
+  OPENQA_API_KEY: ${{ secrets.OPENQA_API_KEY }}
+  OPENQA_API_SECRET: ${{ secrets.OPENQA_API_SECRET }}
+  GH_REPO: ${{ github.event.pull_request.head.repo.full_name }}
+  GH_REF: ${{ github.event.pull_request.head.ref }}
+
+jobs:
+  trigger_and_monitor_openqa:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/containers/tumbleweed:client
+    steps:
+      - uses: actions/checkout@v3
+      - name: Determine the latest Tumbleweed build on o3
+        id: latest_build
+        run: >-
+          echo build=$(curl -s
+          ${OPENQA_HOST:-https://openqa.opensuse.org}/group_overview/${OPENQA_BUILD_LOOKUP_GROUP_ID:-1}.json
+          | jq -r '([ .build_results[] | select(.tag.description=="published") | select(.version=="Tumbleweed") | .build ] | sort | reverse)[]'
+          | head -n1) >> "$GITHUB_OUTPUT"
+      - name: Trigger and monitor a basic openQA test on o3
```

Comment:
for CI stuff: I would consider a code smell if we're using to do something that should be provided by an API.

Reply:
Well, it is provided by `openqa-cli` and that is of course using the CLI internally. I'm not sure what you're getting at exactly.

Reply:
I guess this is about using jq to determine the build? Rather than something like `https://openqa.opensuse.org}/group_overview/${OPENQA_BUILD_LOOKUP_GROUP_ID:-1}.json?type=published` where `type=published` would give us that

Reply:
@foursixnine so what does that mean for this PR? Do you think we need to implement and deploy that feature for openQA first before you would accept this PR or is it a general remark to keep in mind for later? If for the first then I am not sure that this is what we should or will plan to do in the upcoming time.

Reply:
I'd leave it as a follow-up. We'll need to see what API makes sense to start with and we use similar logic elsewhere, too.

Reply:
> @foursixnine so what does that mean for this PR? Do you think we need to implement and deploy that feature for openQA first before you would accept this PR or is it a general remark to keep in mind for later? If for the first then I am not sure that this is what we should or will plan to do in the upcoming time.

I thought you peeps had already done something with this PR, as mentioned before, I won't block a merge anymore.

> is it a general remark to keep in mind for later

It is a general remark for all of the ecosystem (repos under os-autoinst), but it isn't the end of the world at this stage.

> We'll need to see what API makes sense to start with and we use similar logic elsewhere, too.

Lets have that conversation early January, I think there's already a note about something remotely similar, in the "various feature requests" ticket that exists already.

Reply:
> I'd leave it as a follow-up. We'll need to see what API makes sense to start with and we use similar logic elsewhere, too.

One of the things I mentioned to Oliver is to gather data of user behavior of openQA (https://matomo.org/ for instance so you don't need to code your own thing)

Reply:
I believe you meant [poo#152939](https://progress.opensuse.org/issues/152939)